tests/test_amcs.py

```diff
@@ -0,0 +1,1053 @@
+# This file is part of ts_mtdome.
+#
+# Developed for the Vera Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import logging
+import math
+import unittest
+
+import pytest
+from lsst.ts import mtdome
+from lsst.ts.mtdome.mock_llc.amcs import (
+    POWER_PER_MOTOR_CRAWLING,
+    POWER_PER_MOTOR_MOVING,
+)
+from lsst.ts.mtdome.mock_llc.mock_motion.azimuth_motion import NUM_MOTORS
+
+logging.basicConfig(
+    format="%(asctime)s:%(levelname)s:%(name)s:%(message)s", level=logging.DEBUG
+)
+
+_MAX_SPEED = math.radians(4.0)
+START_TAI = 10001.0
+START_MOTORS_ADD_DURATION = 5.5
```

Comment:
Please document _MAX_SPEED and START_MOTORS_ADD_DURATION with code comments.

Also...why the leading underscore in _MAX_SPEED? None of the attributes of this module are public, since it's not library code.

Reply:
All done.